src/python/test/test_dxclient.py

```diff
@@ -76,6 +76,14 @@ def list_folder(proj_id, path):
     return output
 
 
+def test_dxpy_import_stdin_none(monkeypatch):
```

Comment:
Thank you for the contribution!

How do you run this test? Also, we put all the tests under Test classes, so you can for example put your unit tests under `TestDXTestUtils`.

Reply:
I used this to run it:

```
% PYTHONPATH=src/python/test pytest src/python/test/test_dxclient.py::TestDXPYImport
================================================ test session starts =================================================
platform darwin -- Python 2.7.15, pytest-3.5.1, py-1.6.0, pluggy-0.6.0
rootdir: /Users/piotr/counsyl/dx-toolkit/src/python, inifile:
collected 1 item

src/python/test/test_dxclient.py .                                                                             [100%]

============================================== 1 passed in 0.57 seconds ==============================================
```

>Also, we put all the tests under Test classes, so you can for example put your unit tests under TestDXTestUtils.

Done. Moved it to `TestDXPYImport`, because it doesn't need `DXTestCase` or anything, just a simple Python object is enough. Also it doesn't seem like it is similar to other methods in `TestDXTestUtils`. Let me know what you think.

Reply:
Thanks @pkaleta!  Did you commit the move and push to the branch?  (I don't see it at first glance :))

Reply:
Good point @geetduggal :) Check out now.